### PR TITLE
Add seafile database configuration section

### PR DESCRIPTION
**Pull request by Lesterpig**
Following https://github.com/haiwen/seafile-server/issues/167#issuecomment-403554578

### CODE REVIEW

File: config/seafile-conf.md [hunk 1]
```diff
@@ -85,33 +95,33 @@ When users upload files in the web interafece, file server assigns an token to a
...
-## Changing MySQL Connection Pool Size
+## Database configuration
 
-When you configure seafile server to use MySQL, the default connection pool size is 100, which should be enough for most use cases. You can change this value by adding following options to seafile.conf:
+The whole database configuration is stored in the `[database]` section of the configuration file, whether you use SQLite, MySQL or PostgreSQL.
 
 ```
 [database]
-......
-# Use larger connection pool
```
Comment (shoeper) [inline config/seafile-conf.md]: would be better to keep the connection pool comment, I think

Comment (Lesterpig) [reply config/seafile-conf.md]: just pushed an update

**Comment by Lesterpig** [comment]
:up: thanks :smiley:

**Comment by freeplant** [comment]
Thanks!

**Pull request merged**